deploy/docker/seqr/wait_for_routes

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+###
+# Waits for network endpoints.  Intended useage is within Kubernetes CronJobs to wait for sidecar availability.
```

Comment:
don't want to be that guy, but `s/useage/usage` 😄

Reply:
❤️ that the typo is in the comment and not the bash